HelpSource/Classes/Pdef.schelp

```diff
@@ -4,6 +4,8 @@ categories:: Libraries>JITLib>Patterns
 related:: Classes/Pdefn, Classes/PdefGui
 
 description::
+Pdef can be used to specify patterns that can be replaced while running. A Pdef can be manipulated through its key. A Pdef can be started, paused, resumed, reset and stopped programmatically. The old pattern and its replacement can automatically crossfade and the time of replacement can be quantized.
```

Comment:
this paragraph and the paragraph below contain duplicate information ("can be replaced while running" ... "can be replaced while playing"). I think you might be able to just get rid of that phrase in the second paragraph.